### PR TITLE
Remove Congratulations screen and navigate to home after send account success

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/142?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* **#142** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/142?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#138** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/138?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#141](https://github.com/0xsend/sendapp/pull/141) <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/141?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#133** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/133?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#132** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/132?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#125** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/125?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#124** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/124?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#123** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/123?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#122** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/122?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#121** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/121?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#118** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/118?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#119](https://github.com/0xsend/sendapp/pull/119) <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/119?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#116** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/116?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#115** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/115?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#114** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/114?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#113** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/113?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/142?utm_source=stack-comment).

File: packages/app/features/auth/onboarding/screen.tsx [hunk 1]
```diff
@@ -52,64 +40,52 @@ export function OnboardingScreen() {
...
 
       <YStack f={3} jc="center" maw={'100%'} space="$4" gap="$4">
-        {sendAccts?.length === 0 ? <OnboardingForm /> : <SendAccountCongratulations />}
+        <OnboardingForm />
       </YStack>
     </YStack>
   )
 }
-export function SendAccountCongratulations() {
+export function SendSecretStore() {
```
Comment (0xBigBoss) [inline packages/app/features/auth/onboarding/screen.tsx:48]: i don't see where your pulling this in. were you going to make it a new screen?

Comment (youngkidwarrior) [reply packages/app/features/auth/onboarding/screen.tsx:48]: Yes, plan is to add it as a modal accessible from the home dashboard hidden by the DEV flag

This PR is just to remove it so we can experience onboarding in all its glory

Comment (0xBigBoss) [reply packages/app/features/auth/onboarding/screen.tsx:48]: I included SEND token balance in the secret shop, it just landed on dev

File: packages/app/features/auth/onboarding/onboarding-form.tsx [hunk 2]
```diff
@@ -90,7 +92,10 @@ export const OnboardingForm = () => {
     if (error) {
       throw error
     }
-    await refetchSendAccounts()
+    //@todo We need to show something to the user when this fails
```
Comment (0xBigBoss) [inline packages/app/features/auth/onboarding/onboarding-form.tsx:95]: Just show this part. Sure it won’t make much sense to the user but it’s much better than our UI becoming unresponsive. 

Something like.  “Sorry something went wrong 😢unable to create two send accounts”

would be totally fine.

**Review by 0xBigBoss — APPROVED** [review]
Let me know if you need help with The merge conflict.

File: packages/app/features/auth/onboarding/onboarding-form.tsx [hunk 3]
```diff
@@ -92,7 +94,13 @@ export const OnboardingForm = () => {
     if (error) {
       throw error
     }
-    await refetchSendAccounts()
+    try {
+      const { data: sendAccts, error: refetchError } = await refetchSendAccounts()
+      if (refetchError) throw refetchError
+      if (sendAccts?.length && sendAccts.length > 0) replace('/')
+    } catch (e) {
+      throw Error(`Sorry something went wrong \n\n ${e}`)
```
Comment (0xBigBoss) [inline packages/app/features/auth/onboarding/onboarding-form.tsx:102]: perfect ty sir

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Mar 3, 6:58 PM EST**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/142).

**Pull request merged**